sample/common/build.gradle

```diff
@@ -64,18 +65,4 @@ kotlin {
     summary = "Common core for SQLDelight sample."
     homepage = "https://github.com/cashapp/sqldelight/tree/master/sample/common"
   }
-}
-
-task iosTest {
```

Comment:
There's one more of these somewhere else to delete. All of the invocations should be replaced with `iosX64Test`

Reply:
Also this one: https://github.com/cashapp/sqldelight/blob/2173c25ff72b24eda1de68ba2671c447157698a5/sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/IntegrationTest.kt#L157